setup.py

```diff
@@ -31,10 +34,13 @@
     classifiers=[
         'Development Status :: 2 - Pre-Alpha',
         'Intended Audience :: Developers',
-        'License :: OSI Approved :: Apache License',
+        'License :: OSI Approved :: Apache Software License',
         'Natural Language :: English',
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
+    ],
+    install_requires = [
```

Comment:
This seems to conflict with `install_requires=requirements,` above?
The version also doesn't match the one from the README

Reply:
Right, I didn't notice that, will remove it, thanks!